apps/extension/src/app/components/launchdarkly-client-id.tsx

```diff
@@ -0,0 +1,24 @@
+import { useMemo } from 'react';
+
+import { Box, styled } from 'leather-styles/jsx';
+
+import { getClientId } from '@app/common/client-id';
+
+export function LaunchDarklyClientId() {
+  const clientId = useMemo(() => getClientId(), []);
```

Comment:
```suggestion
const clientId = getClientId();
export function LaunchDarklyClientId() {
```

useMemo and useCallback are a plague

Reply:
Thanks. I don't plan to merge this, it's just a quick way to get the client Ids